### ISSUE TITLE
Créer une ontologie Code Social

### ISSUE THREAD

**Issue by Guitouuu** [assemblee-virtuelle/pair#12]
A partir de ce doc : https://docs.google.com/document/d/1Bso65wJHjvJr9A0uo4jR5N52dZXWqy_dxHBipVpDZ0I/edit?usp=sharing

**Comment by Alice-Po** [comment]
A terme on aimerait voir émerger un Tinder des organisation. Et on aimerait s'appuyer sur PAIR. mais dans ce Tinder pour bien faire matcher des entitées, on doit pouvoir exprimer leur identité pour être sur qu'elles soient vraiment compatible.

Par exemple, les Idées rattachées à un Acteurs peuvent être vecteur d'identité.

Pour cette réflexion on oublie ce qu'est ce code social actuel et on le pense par sa raison d'être première : Identifier les compatibilités et les éventuelles problématiques de coopération (technologie : what'sapp vs mail ou idéologique : anarchiste et conservateur).

Il faudrait intégrer le code social non pas comme quelque chose qu'on pourrait pluguer mais par le design.

On pourrait imaginer un questionnaire au terme du quel on sonde la pertinence du modèle mais aussi et surtout ce questionnaire fournit in fine le code sociale de l'orga. Cela deviendrait une bonne pratique pour s'assurer de sa singularité, sa valeur ajoutée au sein de l'écosystème et la pertinence de sa raison d'être. Un rythme initiatique durant le quel tu enregistre pour la première fois ton identité sur le web.

Vous en pensez quoi ?

**Comment by Alice-Po** [comment]
@GuillaumeAV Encore une fois je pense que sous ce format que tu propose, le questionnaire sera inaccessible au commun des mortels.. Je pense qu'il pourrait être plus subtile, plus synthétique, plus inspirant. J'ai l'impression qu'extraire le sens d'une entité à travers les modèles, c'est essayer de faire rentrer des rond dans des carrés..

**Comment by GuillaumeAV** [comment]
haha aah oui tu te réfères à cette issue que j'avais créée en 2016 :) Oui en effet, l'approche que tu proposes, où le code social est, à travers un questionnaire déduit / inferré plutot que déclaré me semble très bonne !! Et effectivement, on peut concevoir un processus avec une forte composante UX / Design / usage ! 

On pourra peut-être par ailleurs s'inspirer de https://www.virtual-assembly.org/open-commons/

**Comment by Alice-Po** [comment]
Est ce qu'on est d'accord de mettre à l'archivage ces 6 modèles pour penser quelques choses de plus parlant ? de plus évocateur ? de plus accessible ?

Pour moi 
- "inspiré par ", fait le job du modèle culturel
- Le modèle écologique est une évidence. Qu'est ce qu'on veut faire resurgir avec ce modèle ? Voir si tu crois à la crise climatique ou pas ? Et Il pourrait donné une donnée non pertinente. Par exemple, le web décentralisé aujourd'hui produit beaucoup de requètes, on est pas du tout écoconception. Peut être qu'il y a des avantages à terme mais on a pas de recul pour se prononcer. Est ce que ca veut dire qu'on fait du web non écolo ? Que dans nos orga cela n'interesse personne ? La réponse est non bien sur ! C'est juste qu'on est focus sur de la R&D d'archi pour l'instant, une chose aprés l'autre..
- Modèle social et humain : Est ce qu'en fait on veut pas expliciter les process de gouvernance plutot ici ? 
- Modèle technologie : Je pense que pour la plupart des organisations cette question est trés difficile à répondre. Cela nécéssite de pouvoir se comparer. On peut se penser hight tech en utilisant du excel. Moi ce que j'ai besoin de savoir c'est si l'argo communique par mail une fois par moi ou par messager tout les jours. Pour le rapport au progrés, à savoir quel est notre référentiel, je pense que cela se place dans la section "inpiration".
- Modèle économique : c'est pertinent, j'ai besoin par exemple de savoir comment se finance l'orga, si elle a des bénévoles etc..
- Modèle juridique : est ce qu'un statut juridique ne fait pas le taff ? Je ne vois pas du tout quoi répondre en plus à ce modèle..

Avec des modèles si larges, on risque de tomber dans des pavés d'évidence et de banalité qui ne permettront pas de faire jaillir la singularité de l'orga.

@GuillaumeAV T'en penses quoi ?

**Comment by GuillaumeAV** [comment]
Une tentative de début de modélisation, à creuser / expliciter !

[Modèle culturel] PAIR **inspiré par** Thing. 
[Modèle économique] PAIR **financé par** Agent - Ressource - Concept 
[Modèle juridique ] PAIR **a pour forme juridique** Concept - Organisation - Document
[Modèle social & humain] **a pour modèle de gouvernance** Concept - Document
[Modèle écolo] PAIR **utilise / produit / consomme** Ressource
[Modèle techno] PAIR **mobilise** Moyen

**Comment by Alice-Po** [comment]
Mieux ! Mais je ne suis pas tout à fait convaincue par le modèle écolo et techno. Et je pense qu'on ne pourra avancer qu'en testant la pertinence sur le terrain.

Donc si on voulait faire un questionnaire pour les orga de terrain, je verrais quelques chose du genre : 

Quelle est la forme juridique de votre organisation ?
Quelles entités ou concepts ont inspirés votre organisation ?
Comment votre organisation est financé ?
Quelle est le modèle de gouvernance de votre structure ? (Document explicatif à joindre ? Pourrait-on imaginer des modèles de gouernance à suggerer ?)
Quelle est la raison d'être en une phrase de votre organisation ? 

Quelles sont les flux de valeur dans votre organisation et quelles sont les parties prenantes ? (Mise en perspective de l'orga au sein de son écosystème, cf: modele ecolo ??)
(Pour cette question, on pourrait donner la possibilité de puiser dans des orga déjà existante dans la base de donnée pour directement créer le lien ?)
Quels sont vos organisations amies, qui partagent vos valeurs ?
Donner 5 mots pour décrire votre orga. (Element essentiel de matching)
Quelle sont les moyens de communication de votre organisation ? (modele techno ??)
Êtes vous affilié à une localisation particulière ? Si oui où ? 

Selon moi, faire ce questionnaire devrait faire réfléchir sur la valeur ajoutée de l'orga au sein de son écosytème et être un moyen de vérifier la pertinence de l'activité. Sans compter sa fonction de référencement pour le fameux Tinder.

T'en penses quoi @GuillaumeAV ?

**Comment by GuillaumeAV** [comment]
Yes c'est top, il faudra veiller à ce que les questions soient précises  pour qu'elles puissent ensuite être traduites en relations et en objets, avec des mécanismes de complétion !
